### DISCUSSION TITLE
小狼毫五笔形码候选显示顺序和码表中的优先级不符

### DISCUSSION THREAD

**Discussion by seszu** [rime/home#1104] [General]
在用小狼毫五笔方案，开启了逐渐提示功能，拿五笔86码表里的：忠顺、中叶、跟踪三个词举例，因为词组在码表里都是4码词条，在按到第三码的时候，这三个词组均出现在候选项中，而且永远是按 忠顺、中叶、跟踪这样的排序，在不按全码的时候它们的排序是不受编码的优先级控制的，而是靠各自的编码来排序的，因为忠顺的第4码为D，中叶的第4码为F，跟踪的第4码为H，所以在不输完整编码的时候，码表里的优先级控制不了他们的排列顺序，不知道这是属于软件自身的问题还是设置的问题。

**Comment by LEOYoon-Tsaw** [comment]
該行爲符合預期

**Reply by seszu** [reply]
> 該行爲符合預期

虽然没有打全编码，但是既然在码表中加了权重，是否应该把权重高的排得靠前一点呢。。。

**Reply by LEOYoon-Tsaw** [reply]
據說編碼不全時也去查權重會十分影響性能。既然都用五筆了，既然都鍵詞了，打全編碼是基本的操作吧？

**Reply by seszu** [reply]
明白了，暂时就把需要三码靠前的新加一个三码的词条吧。谢谢
